policies/privacy-policy.md

```diff
@@ -0,0 +1,96 @@
+    - [Data for Foundation operations](#data-for-foundation-operations)
+- [How We Use Your Information](#how-we-use-your-information)
+- [Information Sharing and Disclosure](#information-sharing-and-disclosure)
+- [How We Protect Your Information](#how-we-protect-your-information)
+- [Data Security and Cross-border Transfers](#data-security-and-cross-border-transfers)
+- [Data Retention](#data-retention)
+- [Your Rights](#your-rights)
+- [Changes to This Policy](#changes-to-this-policy)
+- [Contact Us](#contact-us)
+- [Additional Information for EEA, UK, and California Users](#additional-information-for-eea-uk-and-california-users)
+
+## Information We Collect
+
+We collect only the data necessary to operate our services effectively.
+
+### Data for Foundation operations
+
+1. **Authentication and Identity Data**: We use GitHub for authentication and collect basic identity data, including your GitHub login and user ID.
+    If you serve on a CF committee, we may display your GitHub login, name, and profile bio publicly.
+    Members may also opt to provide an alternate display name or bio.
+
+2. **Email Forwarding Data**: For members using the ForwardEmail service, we link your GitHub login to the forwarding email address(es) specified.
+    ForwardEmail stores the target address; CF does not retain this information.
+
+3. **Session Cookies and Analytics**: Our member section uses temporary session cookies solely for GitHub authentication.
+    We also collect anonymous, aggregated analytics to improve website performance and usability.
+
+4. **Legal and Contributor Data**: For legal agreements (e.g., asset transfers, fiscal hosting) and contributor verification (e.g., Contributor License Agreements or commit messages), we collect names, contact details, and any relevant contribution history.
+
+## How We Use Your Information
+
+We use your information to:
+
+- **Authenticate Access**: GitHub data is used to verify and provide secure access to CF services.
+- **Public Display for Committees**: We publicly display committee members’ names and GitHub logins during their tenure.
+- **Communication**: The ForwardEmail service facilitates communication via forwarding addresses provided by members.
+- **Website Improvement**: Analytics help us assess website performance without identifying individual users.
+- **Project and Contribution Oversight**: Contributor information supports project management and monitors adherence to contribution requirements, such as CLAs or DCOs.
+
+## Information Sharing and Disclosure
+
+We do not sell or rent your personal information.
+Information may be shared under the following circumstances:
+
+- **Third-party Services**: We share your information with third-party services only when you opt-in, such as by using ForwardEmail, and only as necessary for those services to function.
+- **Legal Requirements**: We may disclose information as required by law or in response to a valid legal request.
```

Comment:
I have seen some privacy policies also indicate what jurisdiction will be used to determine if you are accepting these requests, because its not always the place the foundation is located.

For example, Law Enforcement in Malaysia may not have any international standing to request something that happened solely on USA cables.

Reply:
If/when that is an issue, I'll defer to my lawyer on that one (and we're incorporated in the state of florida, which is noted in many places.. ). Not sure I need to repeat .